src/components/MapPanel.vue

```diff
@@ -937,33 +928,228 @@ export default {
     }
 
     .non-mobile-corner > div > div > div> a {
-      width: 45px !important;
+      width: 230px !important;
       height: 45px !important;
     }
 
     .mobile-corner > div > div > div > .leaflet-draw-draw-polygon {
       background-position: -73px -15px !important;
     }
 
-    // HIDES THE INSTRUCTIONS IF MOBILE
-    .leaflet-draw.leaflet-control {
-      clear: unset;
-      float: right;
+    .leaflet-touch .leaflet-draw-actions {
+      left: 230px;
     }
 
-    .leaflet-draw-toolbar.leaflet-bar.leaflet-draw-toolbar-top {
-      width: 49px;
-      height: 49px;
+    .leaflet-bar button {
+      padding: inherit !important;
     }
 
-    .leaflet-touch .leaflet-draw-actions {
-      left: 60px;
+
+          //CSS for search buttons
+
+    .leaflet-bar.leaflet-draw-toolbar>a.leaflet-draw-draw-polygon {
+      border-radius: 0;
     }
 
-    .leaflet-bar button {
-      padding: inherit !important;
+    .leaflet-bar.leaflet-control.buffer-control {
+      button {
+        border-radius: 0;
+      }
+      :focus {
+        outline: none;
+      }
+
     }
+    .leaflet-control-container>div{
+      width: 100%;
+      // .leaflet-control{
+      //   width: inherit;
+      // }
+      .pvm-container-non-mobile {
+        &:hover:after {
+          height: 100%;
+          width: 100%;
+          min-width: 315px;
+          position: absolute;
+          left: 295px;
+          content: "Type an address, owner name, property account number, or Department of Records registry map number.";
+          background: #d3d3d3;
+          opacity: 0.7;
+          display: flex;
+          // align-items: center;
+          color: #000;
+          font-style: italic;
+          padding: 4px;
+          // width: 183%;
+        }
+      }
+    }
+
+    div.buffer-control.leaflet-bar.inactive-buffer-button {
+      &:hover:after {
+        content: "Select a point on the map to show all parcels within 250-foot radius.";
+        background: #d3d3d3;
+        opacity: 0.7;
+        display: flex;
+        // align-items: center;
+        color: #000;
+        font-style: italic;
+        padding: 4px;
+        width: 183%;
+      }
```

Comment:
I think the padding needs to be adjusted - I believe it should be the same for hovering / tooltip as it is when the tooltip is actually selected. See below for differences: 
Hover:
![Screen Shot 2020-04-30 at 10 42 10 AM](https://user-images.githubusercontent.com/1517503/80724043-5ceb2880-8acf-11ea-858c-c4fbbdff3ddd.png)

Selected: 
![Screen Shot 2020-04-30 at 10 43 09 AM](https://user-images.githubusercontent.com/1517503/80724074-6a081780-8acf-11ea-88e9-b2d9f391d780.png)

I might also increase opacity to 0.8 but that's a question for @sarahall. Finally, it's weird that this is italicized but the second is not. I think we should be standardizing these.

Reply:
@danlopez agree, to increase the opacity to 0.8. Also agree to standardize the text style, remove the italics, they both can be regular. 
The text in the Hover State "Select a point on the map to show all parcels within a 250-foot radius." Should replace the text in the Selected state. This should be done for the other hover/selected states as well. The "Cancel" should remain as it is.

Reply:
Are you should we should keep the hover text when selected too?  Or should there be something different when its select?

I'm not sure if replacing the text in the selected state might push the cancel button off screen or conflicting with other elements in some instances  It also seems a little confusing because when it is selected, not much would change except for the subtle addition of the cancel button that is clickable. 

Also, the draw tool text should change when selected bc there are 3 buttons in that box that have functionality to help with using the tool. Will it be confusing or inconsistent to have the buffer tool remain more the same in either state, while the draw tool will change when selected?